### PR TITLE
Add Keyboard navigation on IconPicker

### LINKED ISSUE #2746: Add Keyboard navigation on IconPicker
## Scope & Context

Recently, we had introduced a re-usable SelectableList component which used in CommandMenu. It provides a simple way to track which item is selected with keyboard.
Right now it only support one dimension (arrow up and down) but we would like to:
- extend it to be 2-dimensions
- use it to add arrow selection on IconPicker

## Current behavior

On the IconPicker, it is not possible to select an icon using the keyboard (arrow up, down, left, right)

## Expected behavior

On the IconPicker, is is possible to select an icon using the keyboard

## Technical inputs

- Improve SelectableList API to support 2 dimensions selectableItemIds (string[] => string[][])
- improve SelectableList API to pass the Hotkeyscope (right now we have hardcoded CommandMenu one)
- use it within IconPicker

### PR DESCRIPTION
**Pull request by gitstart-twenty**
Fixes #2746

### CODE REVIEW

**Comment by gitstart-twenty** [comment]
> @gitstart-twenty Please use SelectableList component and its API to implement it :)

Alright

**Review by charlesBochet — COMMENTED** [review]
@gitstart-twenty I have left comment about performance. Other than that, looks good to me

**Comment by gitstart-twenty** [comment]
@charlesBochet We're on this

File: front/src/modules/command-menu/components/CommandMenuItem.tsx [hunk 1]
```diff
@@ -46,7 +51,14 @@ export const CommandMenuItem = ({
...
     }
-  };
+  }, [navigate, onClick, to, toggleCommandMenu]);
+
+  useEffect(() => {
+    if (enterClicked && isSelectedItemId) {
+      onItemClick();
+      resetEnterClicked?.();
+    }
+  }, [enterClicked, isSelectedItemId, onItemClick, resetEnterClicked]);
```
[the remark this answers is not in the document]
Comment (charlesBochet) [reply front/src/modules/command-menu/components/CommandMenuItem.tsx:61]: Taking a look

**Comment by charlesBochet** [comment]
@gitstart-twenty I'm about to rework the frontend folder architecture. I'm merging this PR as it's adding a feature on IconPicker.
Could you please open another PR to refactor CommandMenu so we can implement the onEnter command line 194? :)

**Pull request merged**